Shared/Models/Message Model/Messageable.swift

```diff
@@ -23,7 +23,7 @@ protocol Messageable {
     var isFromCurrentUser: Bool { get }
     var authorId: String { get }
     var attributes: [String: Any]? { get }
-    var avatar: Avatar { get }
+    var avatar: Avatar? { get }
```

Comment:
I made a mock message object for SwiftUI testing. The mock doesn't have an avatar to show so this is optional.